pyproject.toml

```diff
@@ -9,7 +9,7 @@ packages = [{include = "nshm_model_graphql_api"}]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-nzshm-model = {git = "https://github.com/GNS-Science/nzshm-model", rev = "pre-release"}
+nzshm-model = {git = "https://github.com/GNS-Science/nzshm-model", rev = "pre-release-0.13.5"}
```

Comment:
there is a 0.13.6 released and published on pypi we can use

Reply:
I've tried that out and created #32 to upgrade to nzshm-model `0.13.6` (there's a breaking change in the library). For some reason, `pre-release-0.13.5` also no longer works to build this (strangely, it looks like it never should have worked). I've now fixed the version at `0.10.6` which is the last version before a breaking change in `nzshm-model` (see #32).